lookbackoptions

```diff
@@ -0,0 +1,128 @@
+import numpy as np
+from datetime import datetime
+import pyarrow as pa
+from qablet_contracts.timetable import TS_EVENT_SCHEMA
```

Comment:
replace this script by an ipynb script, just like 02/03a/03b. perhaps call it 04_Lookbacks.